src/components/UpdateInfo.vue

```diff
@@ -21,10 +21,44 @@ export default defineComponent({
     const infos = ref<UpdateInfo[]>();
     store.dispatch("GET_UPDATE_INFOS").then((obj) => (infos.value = obj));
 
+    const currentVersion = ref("");
+    window.electron.getAppInfos().then((obj) => {
+      currentVersion.value = obj.version;
+    });
+
+    const latestVersion = ref("");
+    fetch("https://api.github.com/repos/VOICEVOX/voicevox/releases")
+      .then((res) => res.json())
+      .then((obj) => {
+        obj.find((item: { prerelease: boolean }) => {
+          item.prerelease === false;
+        });
```

Comment:
`{}`で囲うとdictが返ってしまうので、なしにしないとバグりそうです･･･！
```suggestion
        obj.find((item: { prerelease: boolean }) => 
          item.prerelease === false;
        );
```

Reply:
修正しました！